Dockerfile.android

```diff
@@ -87,11 +95,11 @@ RUN patch -p1 < ../libev-4.19-android.patch && \
     make install
 
 WORKDIR /root/build
-RUN curl -L -O http://zlib.net/zlib-1.2.8.tar.gz && \
-    tar xf zlib-1.2.8.tar.gz && \
-    rm zlib-1.2.8.tar.gz
+RUN curl -L -O https://downloads.sourceforge.net/project/libpng/zlib/$ZLIB_VERSION/zlib-$ZLIB_VERSION.tar.gz && \
```

Comment:
Could you tell me why zlib.net is not used?  I think it is official.

Reply:
Yes, this is the official version. But officials seem to not store older versions here.

When new versions are released (1.2.11), the old version (1.2.8) will be deleted.

Sourceforge.net is a official produced mirror and contains outdated versions.

Of course, also can be replaced with the official git repo to get this library.

However, if you know the storage location of the old versions on the zlib.net, I'm going to modify it.

Reply:
Thank you.  I realized that zlib.net links to sf.net/libpng.  No problem.